pkgs/build-support/dotnet/fetchnuget/default.nix

```diff
@@ -4,16 +4,20 @@ attrs @
 { pname
 , version
 , url ? "https://www.nuget.org/api/v2/package/${pname}/${version}"
+, hash ? ""
+, sha512 ? ""
```

Comment:
```suggestion
```
If we have sri support we don't need explicit sha512 support. This also reduces code complexity.